Documentation/ReleaseNotes.md

```diff
@@ -137,13 +139,46 @@ For this reason the way we display constraints in the property inspector changed
 The constraints that are applied to the component are now shown as a list in the constraint manager component whereas the component using the constraint manager (either [bounds control](README_BoundsControl.md#constraint-system) or [object manipulator](README_ObjectManipulator.md#constraint-manager)) will now show the selected constraint manager and mode (auto or manual). 
 For more information read the [constraint manager](README_ConstraintManager.md) section in our docs.
 
-### Breaking changes
+###  HoloLens 2 Button material update
+Updated HoloLens 2 Button's front cage material to remove black color in MRC.
+
+![HoloLens 2 button material update](https://user-images.githubusercontent.com/13754172/94341269-dcf7c900-0042-11eb-9028-e55abd2ead67.png)
+
+### Description panel update, movable example scene
+Updated description panel. (SceneDescriptionPanelRev.prefab) New design provides a grabbable top bar which allows the user to adjust/move the entire scene.
+
+![Description panel update](https://user-images.githubusercontent.com/13754172/91176366-28a21480-e71d-11ea-9e80-7e219595de9c.png)
+
+### Spatial mesh visualization - Pulse on air-tap
+Updated pulse shader example for the spatial mesh to match HoloLens 2's shell behavior.
+
+![Pulse on air-tap](https://user-images.githubusercontent.com/13754172/90310153-d0536180-df29-11ea-939a-e9572d4f5670.gif)
+
+### Elastic system (Experimental)
+The Elastic simulation system allows you to build fluid UI and interaction feedback. It includes a wide variety of extensible and flexible subclasses, offering bindings for 4-dimensional quaternion springs, 3-dimensional "volume springs", and simple linear spring systems. 
 
-**Assembly Definition Files Changes**
+<img src="https://user-images.githubusercontent.com/5544935/88151572-568cba00-cbaf-11ea-91c2-d6b51829b638.gif" width="38%">
+<img src="https://user-images.githubusercontent.com/5544935/88151578-58567d80-cbaf-11ea-8f96-d24f2cf0d6e9.gif" width="45.7%">
```

Comment:
I've already added a section about the elastics in a different PR - could you merge those two sections together @cre8ivepark